build/pkgs/pyproject_metadata/type

```diff
@@ -1 +1 @@
-standard
+optional
```

Comment:
Why switched to "optional"?

It may help to add a comment like https://github.com/sagemath/sage/pull/38219/commits/d8907460782d73bc939bb92e27ef3976ac2680d2

Reply:
A mistake, will fix